src/components/dex/Pair.vue

```diff
@@ -183,6 +191,10 @@ export default {
           color: $red;
         }
       }
+
+      .row.active {
+        background: lighten($dark-purple, 10%);
+      }
```

Comment:
I have some active row styling in the simpleTable component already so you'll probably wanna remove that. This is def a more flexible solution.